plugins/jobtype/src/azkaban/jobtype/HadoopSparkJob.java

```diff
@@ -175,6 +188,35 @@ public HadoopSparkJob(String jobid, Props sysProps, Props jobProps, Logger log)
     }
   }
 
+  /**
+   * Add additional namenodes specified in the Spark Configuration
+   * ({@link #SPARK_CONF_ADDITIONAL_NAMENODES}) to the Props provided.
+   * @param props Props to add additional namenodes to.
+   * @see HadoopJobUtils#addAdditionalNamenodesToProps(Props, String)
+   */
+  void addAdditionalNamenodesFromConf(Props props) {
```

Comment:
We cannot. HadoopSecureSparkWrapper runs inside of a separate process which has had its classpath set properly by HadoopSparkJob. HadoopSparkJob, on the other hand, runs inside of the Azkaban process and shares the same classpath. The Azkaban main classpath does not include Spark JARs so I cannot access e.g. the SparkConf class.